DFC.App.DiscoverSkillsCareers.Services/DataProcessors/BaseDataProcessor.cs

```diff
@@ -0,0 +1,41 @@
+﻿using DFC.App.DiscoverSkillsCareers.Models.Assessment;
+using System;
+using System.Linq;
+
+namespace DFC.App.DiscoverSkillsCareers.Services.DataProcessors
+{
+    public class BaseDataProcessor
+    {
+        protected BaseDataProcessor()
```

Comment:
Remove obsolete constructor.  No need to have it when it does nothing.

Reply:
This is sonar cloud insisting that this goes in.